loader/src/main.py

```diff
@@ -71,6 +71,8 @@ def process_event(msg):
         :param msg:
         :return:
         """
+        logger.debug(msg.value())
```

Comment:
don't we have a bug in the logs when the instance of msg.value() is of type KafkaErrorSomething ?

Reply:
I think this issue happens in `manage_kafka_error`, not when we receive events from other services.
I tried something to avoid that